CleverTapSDK/InApps/CustomTemplates/SystemAppFunction/CTSystemTemplateActionHandler.m

```diff
@@ -27,4 +29,21 @@ - (void)promptPushPermission:(BOOL)fbSettings {
     [pushPrimerManager promptForOSPushNotificationWithFallbackToSettings:fbSettings andSkipSettingsAlert:YES];
 }
 
+#pragma mark Open Url System App Function
+
+- (void)handleOpenURL:(NSString *)action {
+    if (action && action.length > 0) {
```

Comment:
If no action, log and return, to make this more readable instead of the if/else.

Reply:
Done